slime/templates/cronjob.yaml

```diff
@@ -0,0 +1,295 @@
+{{- if .Values.cronJob.enabled -}}
+{{- $root := . }}
+# https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#cronjob-v1-batch
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "slime.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  {{- with .Values.cronJob.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+  labels:
+    {{- include "slime.labels" . | nindent 4 }}
+    {{- with .Values.cronJob.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+# https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#cronjobspec-v1-batch
+spec:
+  {{- with .Values.cronJob.concurrencyPolicy }}
+  concurrencyPolicy: {{ . }}
+  {{- end }}
+  {{- with .Values.cronJob.failedJobsHistoryLimit }}
+  failedJobsHistoryLimit: {{ . }}
+  {{- end }}
+  schedule: "{{ .Values.cronJob.schedule }}"
+  {{- with .Values.cronJob.startingDeadlineSeconds }}
+  startingDeadlineSeconds: {{ . }}
+  {{- end }}
+  {{- with .Values.cronJob.successfulJobsHistoryLimit }}
+  successfulJobsHistoryLimit: {{ . }}
+  {{- end }}
+  {{- with .Values.cronJob.suspend }}
+  suspend: {{ . }}
+  {{- end }}
+  # Available in 1.27 or later
+  # https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-job-limitations
+  {{- with .Values.cronJob.timeZone }}
+  timeZone: {{ . }}
+  {{- end }}
+  # https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#jobtemplatespec-v1-batch
+  jobTemplate:
+    metadata:
+      annotations:
+        checksum/configmap: '{{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}'
+        checksum/secret: '{{ include (print $.Template.BasePath "/secret.yaml") . | sha256sum }}'
```

Comment:
This parameter is usually used to trigger a pod replacement in `Deployment`. Therefore, it may not be necessary in a CronJob. However, there is no problem with including it.

Reply:
Thank you for the reply.

I removed the checksum in this commit.

https://github.com/chatwork/charts/pull/286/commits/675b0f05f73fb766810108c35acf5af375839170